packages/server/lib/config.coffee

```diff
@@ -594,18 +618,28 @@ module.exports = {
     Promise
     .try ->
       ## resolve full path with extension
-      obj.pluginsFile = require.resolve(pluginsFile)
+      obj.pluginsFile = utils.resolveModule(pluginsFile)
       debug("set pluginsFile to #{obj.pluginsFile}")
     .catch {code: "MODULE_NOT_FOUND"}, ->
-      debug("plugins file does not exist")
+      debug("plugins module %s does not exist", pluginsFile)
       if pluginsFile is path.resolve(obj.projectRoot, CONFIG_DEFAULTS.pluginsFile)
-        debug("plugins file is default, check if #{path.dirname(pluginsFile)} exists")
+        debug("plugins file %s is default, check if folder %s exists",
+          pluginsFile, path.dirname(pluginsFile))
+
         fs.pathExists(pluginsFile)
         .then (found) ->
           if found
-            debug("plugins folder exists, set pluginsFile to false")
-            ## if the directory exists, set it to false so it's ignored
-            obj.pluginsFile = false
+            debug("is there index.ts in the plugins folder %s?", pluginsFile)
```

Comment:
can you try to consolidate these debug lines so we don't need a bunch of them scattered everywhere? we are logging out the same data across many modules already and i want to decrease duplication in debug logs as much as possible

Reply:
I don't think the number of debug log messages is a problem, especially compared to debugging production code - then seeing each branch taken is huge benefit

Reply:
agreed, but you can accomplish the same thing with fewer debug lines by logging out branch conditions instead of logging out a different line for each branch, stuff like that... plus, pluginsFile is logged out prior to this point by other config code